spinalcordtoolbox/scripts/sct_deepseg.py

```diff
@@ -375,7 +416,7 @@ def main(argv: Sequence[str]):
             iterator = zip(input_filenames, output_filenames[0::2], output_filenames[1::2])
 
         # Create one QC report per input image, with one or two segs per image
-        species = 'mouse' if any(s in arguments.task[0] for s in ['mouse', 'mice']) else 'human'  # used for resampling
+        species = 'mouse' if any(s in arguments.task for s in ['mouse', 'mice']) else 'human'  # used for resampling
```

Comment:
Since the model names got standardized to `mouse` instead of `mice`, this test can be simplified (and here the substring test is relevant!):
```suggestion
        species = 'mouse' if 'mouse' in arguments.task else 'human'  # used for resampling
```